roles/container_runtime/defaults/main.yml

```diff
@@ -71,10 +71,59 @@ docker_no_proxy: "{{ openshift.common.no_proxy | default('') }}"
 openshift_use_crio: False
 openshift_use_crio_only: False
 
+l_openshift_image_tag_default: "{{ openshift_release }}"
+l_openshift_image_tag: "{{ openshift_image_tag | default(l_openshift_image_tag_default) | string}}"
 
+# --------------------- #
+# systemcontainers_crio #
+# --------------------- #
 l_insecure_crio_registries: "{{ '\"{}\"'.format('\", \"'.join(l2_docker_insecure_registries)) }}"
 l_crio_registries: "{{ l2_docker_additional_registries + ['docker.io'] }}"
 l_additional_crio_registries: "{{ '\"{}\"'.format('\", \"'.join(l_crio_registries)) }}"
 
-l_openshift_image_tag_default: "{{ openshift_release }}"
-l_openshift_image_tag: "{{ openshift_image_tag | default(l_openshift_image_tag_default) | string}}"
+openshift_crio_image_tag_default: "latest"
+
+l_crt_crio_image_tag_dict:
+  openshift-enterprise: "{{ l_openshift_image_tag }}"
+  origin: "{{ openshift_crio_image_tag | default(openshift_crio_image_tag_default) }}"
+
+l_crt_crio_image_prepend_dict:
+  openshift-enterprise: "registry.access.redhat.com/openshift3"
+  origin: "docker.io/gscrivano"
+
+l_crt_crio_image_dict:
+  Fedora:
+    crio_image_name: "cri-o-fedora"
+    crio_image_tag: "latest"
+  CentOS:
+    crio_image_name: "cri-o-centos"
+    crio_image_tag: "latest"
+  RedHat:
+    crio_image_name: "cri-o"
+    crio_image_tag: "{{ openshift_crio_image_tag | default(l_crt_crio_image_tag_dict[openshift_deployment_type]) }}"
```

Comment:
could we simply provide the full image name here (except for RedHat where there is also the tag that changes)?  The docker.io images are temporary here until we have a better place for them.

In any case, this is not blocking for me as it doesn't depend on this PR, just pointing it out that later we might revisit this part.

Reply:
Yeah, I think that's a fine idea.  I wasn't sure what exactly is required for these images, seems like the variables and how we set them could use a redo.